src/main/react/package.json

```diff
@@ -0,0 +1,165 @@
+{
+  "name": "working_hours_plugin",
+  "version": "1.0.0",
+  "description": "React Project for Working Hours Plugin",
+  "repository": {
+    "type": "git"
+  },
+  "engines": {
+    "npm": ">=5",
+    "node": ">=8.10.0"
+  },
+  "author": "Jack Shen",
+  "license": "MIT",
+  "scripts": {
```

Comment:
Are you using all these? Seems like a lot of scripts, but if you find them useful, it's fine with me

Reply:
No, I guess, these are all from the boilerplate that I chose, and I'll check them all and remove the unnecessary ones.